app/main.py

```diff
@@ -1,6 +1,39 @@
+import os
+import requests
+
+
+API_KEY = os.getenv("WEATHER_API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    if API_KEY is not None:
+        payload = {
+            "key": API_KEY,
+            "q": FILTERING,
+        }
+        res = requests.get(BASE_URL + "/current.json", params=payload)
+        try:
+            res.raise_for_status()
+        except requests.exceptions.HTTPError as err:
+            print(f"Error: {err}")
+            return
+        data = res.json()
+        location = data.get("location")
+        current = data.get("current")
+        if location is not None or current is not None:
```

Comment:
The condition `if location is not None or current is not None:` should use 'and' instead of 'or'. This ensures that both `location` and `current` data are present before proceeding. Using 'or' might lead to errors if one of them is `None`.